dm/pkg/schema/tracker.go

```diff
@@ -78,9 +78,8 @@ type downstreamTracker struct {
 
 // DownstreamTableInfo contains tableinfo and index cache.
 type DownstreamTableInfo struct {
-	TableInfo            *model.TableInfo   // tableInfo which comes from parse create statement syntaxtree
-	AbsoluteUKIndexInfo  *model.IndexInfo   // absolute uk index is a pk/uk(not null)
-	AvailableUKIndexList []*model.IndexInfo // index list which is all uks
+	TableInfo        *model.TableInfo // tableInfo which comes from parse create statement syntaxtree
+	WhereHandleCache *sqlmodel.WhereHandle
```

Comment:
why call this cache ?

Reply:
it's computed once for many row changes

Reply:
the whole struct is cached in `tr.dsTracker.tableInfos` which should be called `xxxCache`.

prefer remove this suffix here.